physionet-django/project/templates/project/data_access_request_table_snippet.html

```diff
@@ -21,6 +21,7 @@ <h2>Previous Requests</h2>
                     <td>{{ req.decision_datetime|date }}</td>
                     <td>{{ req.data_use_title }}</td>
                     <td>{{ req.data_use_purpose | safe }}</td>
+                    <td>{% if req.lay_summary %}{{ req.lay_summary | safe }}{% endif %}</td>
```

Comment:
I think in general, even if data is not provided I think it is better to display the field to the user (so that they know it was not provided). Otherwise they have a form that changes based on the application.